### PR TITLE
Completed express-get-json exercise.

### PR DESCRIPTION
**Pull request by sharon-tieu**
Screen Shot 2022-10-17 at 12 06 03 PM

### CODE REVIEW

File: express-get-json/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  // eslint-disable-next-line no-console
+  console.log(`listening on port ${port}~`);
+});
+
+app.get('/api/grades', function (req, res) {
+  const data = [];
+  for (const key in grades) {
+    data.push(grades[key]);
+  }
+  console.error(data);
```
Comment (scbowler) [inline express-get-json/index.js]: Here we want to send the array of grades back to the client, not just console log it.

```JAVASCRIPT
res.json(data);
```

**Review by scbowler — CHANGES_REQUESTED** [review]
Heading in the right direction, just one update needed. Let me know if you have any questions.

File: express-get-json/index.js (hunk 1 above)
Comment (sharon-tieu) [reply express-get-json/index.js]: I see! Thank you for the feedback! That makes sense.

**Comment by sharon-tieu** [comment]
Screen Shot 2022-10-17 at 12 30 41 PM

**Review by scbowler — APPROVED** [review]
Great job implementing the feedback, looks good! 👍

**Pull request merged**